strongbox-web-core/src/main/java/org/carlspring/strongbox/controller/NugetPackageController.java

```diff
@@ -60,32 +77,32 @@
     @ApiOperation(value = "Used to check storage availability")
     @ApiResponses(value = { @ApiResponse(code = HttpURLConnection.HTTP_OK, message = "Storage available."),
                             @ApiResponse(code = HttpURLConnection.HTTP_UNAUTHORIZED, message = "Storage requires authorization.") })
-    @RequestMapping(path = { "{storageId}/{repositoryId}", "greet" }, method = RequestMethod.GET)
+    @RequestMapping(path = { "{storageId}/{repositoryId}/", "greet" }, method = RequestMethod.GET)
```

Comment:
You a right about this method, because I have checked it again and found that Nuget send this request without trailing `/`.
But in case of upload request, then Nuget sends it exactly using trailing `/`, and mapping without it not working